src/plugins/discover/public/url_generator.ts

```diff
@@ -0,0 +1,114 @@
+/*
```

Comment:
dear @streamich I think since more and more file start to populate the `public/` folder it might be a good time to start a `public/utils` folder with the url_generate as it's first happy inhabitant? wdyt?

Reply:
I can create a `utils` folder, but, in general, I thought there was an agreement in Kibana to avoid using general utility folders like `/utils` and `/lib`.